src/Serilog.Sinks.Seq/Sinks/Seq/HttpLogShipper.cs

```diff
@@ -280,6 +282,58 @@ void OnTick()
             }
         }
 
+        private const string _invalidPayloadFilePrefix = "invalid-";
```

Comment:
The codebase doesn't use explicit `private` modifiers at all. Constants are also PascalCased in this solution. Just `const string InvalidPayloadFilePrefix` would be great here. A few explicit `private`s below also.